test/Parse/closure_params.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-swift-frontend -typecheck -verify %s
+
+struct Bar {}
+
```

Comment:
I’ve committed this test [already](https://github.com/apple/swift/blob/5a1cae119ef283508f7a3218d251639a88196822/test/decl/func/functions.swift#L190).  Could you rerun the test suite and update the diagnostics there?

Reply:
Done